Fix/issue#14 desabilitar select

thfields opened the pull request:
desabilita o select do ano e museu em retificar.

thfields committed:
fix: desabilita o select parcialmente, mas ainda tem um bug que nao desabilita total

thfields committed:
docs: comentando o select desabilitado

desabilita o select na pagina de retificar

Everbero commented on src/components/Uploader.tsx:
```diff
@@ -301,6 +305,7 @@ const Uploader: React.FC<{
               <Select
```
não poderia usar algo como {{ isRetificar && <JSX> }} aqui em vez de só desabilitar o select?

thfields replied:
sim, inclusive corrige inserindo uma div que bloqueia o select por completo, pois o componente tem o bug que só desabilita parcialmente. o commit foi [7f32128](https://github.com/Nocs-lab/inbcm-public-frontend/commit/7f32128d2377040505d489cf527edefebd1273f4)

Everbero replied:
@thfields não seria melhor fazer algo assim? Se o select vai estar desativado é não exibi-lo, pq se renderizar o componente não daria para mudar o html dele pelo inspetor? Assim não precisaria adicionar esta div que vc colocou em volta.

```js
    {{ !isRetificar && <Select .../>}}
```

thfields committed:
fix: inserindo bloqueio do select por completo

fix Issue#14

thfields committed:
fix: issue#14-removendo selects do retificar e adicionando o ano e museu como textos

remove o select do ano e do museu do upload quando é chamado pelo retificar, insere o ano e o museu
como texto no retificar e altera o botao de voltar para ir até a página da declaração

issue#14

thfields commented:
fiz a alteração que pediu, quando o uploader é aberto pelo retificar, o select do ano e do museu não são carregados. Em compensação, inseri no componente Retificar o id da declaração e o ano com o museu embaixo como textos para o usuário ficar identificando qual a declaração que está sendo retificada. também alterei o caminho do botão de "voltar" pois voltando para a tela inicial estava dificultando o trabalho e seria ideal que ele voltasse para tela anterior (o index da declaração) e sabendo que já tem o botão "Cancelar" que já vai até a página inicial. o commit foi: e4fa6f65ddf44d89d504c815054e4c9ab00170a6

Everbero commented:
Closes #14
